<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flight22</class>
 <widget class="QWidget" name="Flight22">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>К стоимости</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>К расписанию</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>79</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rad1">
      <property name="text">
       <string>Станция 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad2">
      <property name="text">
       <string>Станция 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad3">
      <property name="text">
       <string>Станция 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad4">
      <property name="text">
       <string>Станция 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>60</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="rad10">
      <property name="text">
       <string>Рейс1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad11">
      <property name="text">
       <string>-Рейс1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad12">
      <property name="text">
       <string>Рейс2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad13">
      <property name="text">
       <string>-Рейс2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad14">
      <property name="text">
       <string>Рейс3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad15">
      <property name="text">
       <string>-Рейс3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>79</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="rad5">
      <property name="text">
       <string>Станция 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad6">
      <property name="text">
       <string>Станция 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad7">
      <property name="text">
       <string>Станция 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad8">
      <property name="text">
       <string>Станция 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Рейс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Конечную остановку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Место посадки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Перейти к оплате</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
